specifications/api/swagger.yaml

```diff
@@ -251,6 +251,13 @@ x-amount: &amount
       enum:
         - lovelace
 
+x-scriptStatus: &scriptStatus
+  type: string
+  enum:
+    - awaits_signing
+    - spent
+    - staked
```

Comment:
I am not sure to understand the rationale behind those statuses? From our point of view, a script address exists on the blockchain or it doesn't. It'd be valuable however to show the balance available at that script address.

Reply:
I expect to have the following lifecycle of addresses with script:
1. some funds are sent to this address
2. even more money can be sent
3. tx with script with all required signatures is sent
 and depending on where script is embedded, we have money spent or money staked
1-2 -> script awaits signing
3 -> funds are unlocked and are spent or staked